app/screens/settings-screen/settings-screen.tsx

```diff
@@ -49,6 +50,42 @@ export const SettingsScreen: ScreenType = ({ navigation }: Props) => {
     }`, { fetchPolicy: "cache-only" }
   )
 
+  const securityAction = () => {
+    var isBiometricsEnabled = null
+    var isPinEnabled = null
+
+    RNSecureKeyStore.get("isBiometricsEnabled")
+      .then((res) => {
+        if (isPinEnabled !== null) {
+          navigation.navigate("security", { mIsBiometricsEnabled: true, mIsPinEnabled: isPinEnabled })
+        } else {
+          isBiometricsEnabled = true
+        }
+      }, (err) => {
+        if (isPinEnabled !== null) {
+          navigation.navigate("security", { mIsBiometricsEnabled: false, mIsPinEnabled: isPinEnabled })
+        } else {
+          isBiometricsEnabled = false
+        }
+      })
+
+    RNSecureKeyStore.get("PIN")
+      .then((res) => {
+        if (isBiometricsEnabled !== null) {
+          navigation.navigate("security", { mIsBiometricsEnabled: isBiometricsEnabled, mIsPinEnabled: true })
+        } else {
+          isPinEnabled = true
+        }
+      }, (err) => {
+        if (isBiometricsEnabled !== null) {
+          navigation.navigate("security", { mIsBiometricsEnabled: isBiometricsEnabled, mIsPinEnabled: false })
+        } else {
+          isPinEnabled = false
+        }
+      })
+
+  }
```

Comment:
Let's make this an util function in case we need this check somewhere else in the future

Reply:
This sounds like pre-mature optimisation.
> _in case_ we need

I think its fine to keep it in the this file as long as its the only place we're using it.

Not a big deal though.